### DISCUSSION TITLE
How to see available state names for a Entity / Sensor on Z-Wave JS

### DISCUSSION THREAD

**Discussion by Salar80** [zwave-js/zwave-js#3568] [Q&A]
After I upgraded from the retired Z-Wave platform to the new Z-Wave JS for many sensors or conditions it does not return numbers anymore and started to return a status like "Closed" "Open" "Clear" "On" or "Off". Is there any way to the table or pam for one device or Sensor?
For Example, Me Smoke Detector has an Entity called "binary_sensor.zcombo_g_smoke_co_alarm_co_alarm_carbon_monoxide_detected   " in it is "Clear"  now, but how should I know what will be the value alarm is active to set an automation for it?

Thanks,
Salar

**Comment by kpine** [comment]
Hi, Home Assistant questions should be asked in the HA [community forums](https://community.home-assistant.io/) or [Discord zwave channel](https://discord.gg/c5DvZ4e).

To answer your question, binary sensor states are either 'on' or 'off'. Use the States page in the Developer tools to see the states each entity reports.
